src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -26,6 +26,13 @@ private static string GetFileNameFromResourceName(string s)
 
         public int CreateEmptyProject(string languageName, string templateDir)
         {
+            // Check if project.json exists in the folder
+            if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "project.json")))
```

Comment:
should "project.json" be a static readonly var here instead of a hardcoded string literal?